Description: Set SOVERSION for the shared library tensorflow-lite
Forwarded: not-needed
Author: Dylan Aïssi <daissi@debian.org>

--- a/tensorflow/lite/CMakeLists.txt
+++ b/tensorflow/lite/CMakeLists.txt
@@ -634,6 +634,9 @@
 )
 add_library(${PROJECT_NAME}::tensorflowlite ALIAS tensorflow-lite)
 
+# Set soversion
+set_target_properties(tensorflow-lite PROPERTIES VERSION 2.14.1 SOVERSION 2.14.1)
+
 # The install targets.
 if(TFLITE_ENABLE_INSTALL)
   install(
